fvm/src/syscalls/env.rs

```diff
@@ -0,0 +1,23 @@
+use super::Context;
```

Comment:
Not sure if `env` is the appropriate module name here. Could also be `chain`.

Reply:
we can rename, let's discuss. I am keeping it for now, but let's resolve before merging.